### PR TITLE
Feature/interactable use

### PR DESCRIPTION
**Pull request by lucasvdiepen**

### CODE REVIEW

File: Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs [hunk 1]
```diff
@@ -31,13 +31,27 @@ public abstract class Interactable : NetworkBehaviour
 
         private protected virtual void Awake() => BoxCollider = GetComponent<BoxCollider>();
 
+        /// <summary>
+        /// A function that returns all colliding colliders.
+        /// </summary>
+        /// <returns>An array of colliding <see cref="Collider"/>'s.</returns>
+        public Collider[] GetCollidingColliders() =>
+            Physics.OverlapBox(BoxCollider.center + transform.position, BoxCollider.size / 2, transform.rotation);
```
Comment (Joost-Robat) [inline Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs:39]: Did you check to see if this is correct size?

**Review by Joost-Robat — COMMENTED** [review]
Looks good. Nice job.

File: Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs (hunk 1 above)
Comment (lucasvdiepen) [reply Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs:39]: I did not check it with a gizmo but it should be correct this way. Everything looks normal when testing.

Comment (Joost-Robat) [reply Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs:39]: Resolved.

**Review by Joost-Robat — APPROVED** [review]
Looks good to me! Ready for merge!

**Pull request merged**